Makefile.am

```diff
@@ -68,6 +68,7 @@ AM_LDFLAGS = $(COV_LDFLAGS) $(ICU_LIBS)
 
 BUILT_SOURCES = \
 	imap/imap_err.c \
+	imap/jmap_err.c \
```

Comment:
I assume this is meant to be `.h`? :)

Reply:
No.  Since BUILT_SOURCES contains imap/imap_err.c I see no reason why it should contain imap/jmap_err.h instead of imap/jmap_err.h.

Reply:
Oh wait, that's my bad, I misread "imap" and "jmap" as being the same, whoops!